README.md

```diff
@@ -59,13 +59,15 @@ dsc_windowsfeature {'IIS':
 }
 ~~~
 
-All DSC Resource names and parameters have to be in lowercase, e.g: `dsc_windowsfeature` or `dsc_name`.
+All DSC Resource names and parameters have to be in lowercase, for example: `dsc_windowsfeature` or `dsc_name`.
 
-When it comes to `ensure =>` (Puppet's `ensure`) versus `dsc_ensure =>` (DSC's `Ensure`), you can use one or the other with the exception of a known issue - `ensure => absent` will report success and do nothing - see [MODULES-2966](https://tickets.puppetlabs.com/browse/MODULES-2966) for details. **Until this issue is resolved, we recommend using `dsc_ensure` exclusively.** Once that issue is resolved you can use one or the other in your manifests for Puppet DSC resource types. If you use both in a Puppet DSC resource, `dsc_ensure` will override the value in `ensure`, so the value for `ensure` will essentially be ignored. It is recommended that `dsc_ensure` is used as it is a closer match for converting from the DSC properties to Puppet DSC resources and it overrides `ensure` so there is less confusion if both are accidentally included.
+> Note: Normally, you can use either `ensure =>` (Puppet's `ensure`) or `dsc_ensure =>` (DSC's `Ensure`) in your manifests. However, there is currently a known issue where `ensure => absent` reports success but does nothing. See [MODULES-2966](https://tickets.puppetlabs.com/browse/MODULES-2966) for details. **Until this issue is resolved, we recommend using `dsc_ensure` exclusively.**
+
+You can use either `ensure =>` (Puppet's `ensure`) versus `dsc_ensure =>` (DSC's `Ensure`) in your manifests for Puppet DSC resource types. If you use both in a Puppet DSC resource, `dsc_ensure` overrides the value in `ensure`, so the value for `ensure` is essentially ignored. It is recommended that `dsc_ensure` is used as it is a closer match for converting from the DSC properties to Puppet DSC resources and it overrides `ensure` so there is less confusion if both are accidentally included.
 
 ### Handling Reboots with DSC
 
-You will need to add the following `reboot` resource to your manifest. It must have the name `dsc_reboot` for the `dsc` module to find and use it.
+Add the following `reboot` resource to your manifest. It must have the name `dsc_reboot` for the `dsc` module to find and use it.
```

Comment:
You are not required to create this, but you can as an optional behavior if you want Puppet to handle reboots when DSC resources request them